src/internal/analytics/components/analytics-funnel.tsx

```diff
@@ -114,7 +113,7 @@ const InnerAnalyticsFunnel = ({ children, stepConfiguration, ...props }: Analyti
   const loadingButtonCount = useRef<number>(0);
   const wizardCount = useRef<number>(0);
   const latestFocusCleanupFunction = useRef<undefined | (() => void)>(undefined);
-
+  const formSubmitStartTime = useRef<number>(0);
```

Comment:
Don't we need to initialize `useRef` with `performance.now()`?

Reply:
yes